gitpod-io/gitpod
# [usage] Move migration file to correct directory

## Description

https://github.com/gitpod-io/gitpod/pull/11439 added a new migration but added it to the top-level of the repo rather than in the migrations subdir. The migration would therefore never be applied. This PR moves the migration to the correct place.

## Related Issue(s)
https://github.com/gitpod-io/gitpod/pull/11439

## How to test

```
cd components/gitpod-db
leeway build .:dbtest-init
```

Then connect to the database and see that the columns added by the migration are present in the `d_b_workspace_instance_usage` table.


## Release Notes

```release-note
NONE
```

## Documentation

## Werft options:
